packages/blocks/src/page-block/edgeless/frame-manager.ts

```diff
@@ -127,3 +125,24 @@ export class EdgelessFrameManager {
     });
   }
 }
+
+export function getBlocksInFrame(
+  page: Page,
+  model: FrameBlockModel,
+  fullyContained: boolean = true
+) {
+  const bound = Bound.deserialize(model.xywh);
+
+  return (
+    page.getBlockByFlavour([
```

Comment:
how about using `surface.getblocks(EdgelessBlockType)`

Reply:
I try not to rely on the surface cause' it does not exist when rendering edgeless content in doc mode.